mpbb-install-dependencies

```diff
@@ -101,34 +101,65 @@ install-dependencies() {
         text="Installing dependency ($dependencies_counter of $dependencies_count) '${depname}' with variants '${depvariants}'"
         echo "----> ${text}"
         echo -n "${text} ... " >> "$log_status_dependencies"
-        # $depvariants isn't quoted on purpose
-        # shellcheck disable=SC2086
-        if ! "${option_prefix}/bin/port" -d install --unrequested "$depname" $depvariants; then
-            echo "Build of dependency '${depname}' with variants '${depvariants}' failed, aborting." >&2
-            echo "[FAIL]" >> "$log_status_dependencies"
-            echo "Building '$port' ... [ERROR] (failed to install dependency '${depname}') maintainers: $(get-maintainers "$port" "${depname}")." >> "$log_subports_progress"
-
-            # Update failcache
-            # $depvariants isn't quoted on purpose
-            # shellcheck disable=SC2086
-            failcache_failure "$depname" $depvariants
-            if [ $? -ne 0 ]; then
-                err "failcache_failure $depname $depvariants failed."
-                return 1
-            fi
-            return 1
-        else
+        # $option_prefix and $thisdir are set in mpbb
+        # shellcheck disable=SC2154
+        if [[ -f $("${option_prefix}/bin/port-tclsh" "${thisdir}/tools/archive-path.tcl" "${depname}" "${depvariants}") ]]; then
```

Comment:
`install-dependencies` is super useful for many reasons:
* it doesn't clutter the logs of port installation (one can concentrate on real problems with port itself rather than browsing through zillion lines of dependencies being built)
* when we set up a new build slave, we don't have any dependencies yet, so some dependencies actually need to be built, not just activated